bear.install

```diff
@@ -27,6 +27,9 @@ function bear_install() {
 function _bear_setup_base_configurations() {
   // Set front page to "node".
   \Drupal::configFactory()->getEditable('system.site')->set('page.front', '/node')->save(TRUE);
+  // Install devel submodules.
```

Comment:
I have no idea if this is the correct file/place to plop these, but they're working!